ui-v2/app/helpers/service/health-checks.js

```diff
@@ -1,6 +1,19 @@
 import { helper } from '@ember/component/helper';
 
-export function healthChecks([item], hash) {
+export function healthChecks([item, proxy], hash) {
+  if (proxy) {
+    switch (true) {
+      case item.ChecksCritical !== 0 || proxy.ChecksCritical !== 0:
+        return 'critical';
+      case item.ChecksWarning !== 0 || proxy.ChecksWarning !== 0:
+        return 'warning';
+      case item.ChecksPassing !== 0 || proxy.ChecksPassing !== 0:
+        return 'passing';
+      default:
+        return 'empty';
+    }
+  }
+
```

Comment:
There's probably some way to avoid the switch duplication here:

```javascript
case item.ChecksCritical !== 0 || (typeof proxy !== 'undefined' && proxy.ChecksCritical !== 0):
        return 'critical';
```

...that way you don't end up with 2 switch statements, but understand its one of those 'many ways to skin a cat' things!

Actually, I wonder if you could make proxy into a default argument?

```javascript
export function healthChecks([item, proxy = {ChecksCritical: 0, ChecksWarning: 0, ChecksPassing: 0}], hash) {
```

Then you don't even need to check if proxy exists (yet another way to skin a cat!)

Completely up to you all that ^.